doc/tutorials/acrn_on_qemu.rst

```diff
@@ -0,0 +1,268 @@
+
+      .. figure:: images/acrn_qemu_1.png
+         :align: center
+
+   b. We use GRUB to boot ACRN, so please install it when prompted.
+
+      .. figure:: images/acrn_qemu_2.png
+         :align: center
+
+3. To login to the SOS guest, find the IP address of the guest to SSH. This can be done via the
+virsh command as shown below,
+
+   .. figure:: images/acrn_qemu_3.png
+      :align: center
+
+4. Once ACRN hypervisor is enabled, the above virsh command might not display the IP. So enable Serial console by,
+
+   .. code-block:: none
+
+      sudo systemctl enable serial-getty@ttyS0.service
+      sudo systemctl start serial-getty@ttyS0.service
+
+5. Enable grub menu to choose between Ubuntu Vs ACRN hypervisor. Modify :file:`/etc/default/grub` and edit below entries,
+
+      .. code-block:: none
+
+         GRUB_TIMEOUT_STYLE=menu
+         GRUB_TIMEOUT=5
+         GRUB_CMDLINE_LINUX_DEFAULT=""
+         GRUB_GFXMODE=text
+
+6. sudo update-grub
+
+7. Once the above steps are done, SOS guest can also be launched using, ``virsh start ACRNSOS --console``. Make sure to use the domain name
+you used while creating the VM instead of ``ACRNSOS``.
+
+This concludes setting up of SOS and preparing it to boot ACRN hypervisor.
+
+
+Install ACRN Hypervisor
+***********************
+
+1. Clone ACRN hypervisor.
+
+   .. code-block:: none
+
+      $ git clone https://github.com/projectacrn/acrn-hypervisor.git
+      $ cd acrn-hypervisor/
+
+2. Use the following command to build ACRN for QEMU,
+
+   .. code-block:: none
+
+      make all BOARD_FILE=../acrn-hypervisor/misc/acrn-config/xmls/board-xmls/qemu.xml  SCENARIO_FILE=../acrn-hypervisor/misc/acrn-config/xmls/config-xmls/qemu/sdc.xml
+
+ For more details, refer to :ref:`getting-started-building`.
+
+3. Copy acrn.32.out from build/hypervisor to SOS guest /boot/ directory.
+
+4. Update the SOS kernel so as to include the virtio-blk driver. UOS (L2 guest) uses virtio-blk
+driver to mount rootfs.
+
+   a. Clone the repo, https://github.com/projectacrn/acrn-kernel.git.
+   b. cd acrn-kernel
+   c. Copy kernel_config_uefi_sos to .config
+   d. make olddefconfig
+   e. make menuconfig
+
+      .. figure:: images/acrn_qemu_4.png
+         :align: center
+
+   f. make
+   g. Copy bzImage go the SOS guest /boot/ directory
+
+5. Update Ubuntu GRUB to boot ACRN hypervisor and load ACRN Kernel Image. Append the following
+configuration to the :file:`/etc/grub.d/40_custom`,
+
+   .. code-block:: none
+
+      menuentry 'ACRN hypervisor' --class ubuntu --class gnu-linux --class gnu --class os $menuentry_id_option 'gnulinux-simple-e23c76ae-b06d-4a6e-ad42-46b8eedfd7d3' {
+         recordfail
+         load_video
+         gfxmode $linux_gfx_mode
+         insmod gzio
+         insmod part_msdos
+         insmod ext2
+
+         echo 'Loading ACRN hypervisor with SDC scenario ...'
+         multiboot --quirk-modules-after-kernel /boot/acrn.32.out
+         module /boot/bzImage_sos Linux_bzImage
```

Comment:
Is there an expectation here that the bzImage` you took above be copied under one of those names? (I.e. `bzImage_sos` or `Linux_bzImage`)

Reply:
We are creating 2 bzImage one for SOS and UOS in the user guide. Here I mean bzImage that is used for SOS, so named it bzImage_sos. Linux_bzImage is the kernel mod tag and should be used as stated in the doc. This should not be changed.